### ISSUE TITLE
Error when using alpine Docker image

### ISSUE THREAD

**Issue by gmolto** [grycap/scar#28]
The execution of the alpine:latest Docker image in AWS Lambda results in the following error:

```
Error: command not found or has no execute bit set:  ['/bin/sh', '/tmp/udocker/script.sh']
Info: creating repo: /tmp/home/.udocker
Warning: running as uid 0 is not supported by this engine
Warning: non-existing user will be created
```

The same Lambda function works well with ubuntu:16.04 and centos:7 

The weird point is that udocker works fine when executing a container out of alpine:latest on a local machine.

**Comment by micafer** [comment]
It seems that it a problem with the libfakechroot used. Using the alpine libfakechroot lib (linked to musl) it seems to work.

**Comment by frezbo** [comment]
@gmolto What if the alpine image I use already has glibc installed https://github.com/grycap/scar/issues/28#issuecomment-304468262

**Comment by gmolto** [comment]
@frezbo Hi. This was already tested unsuccessfully with the frolvlad/alpine-glibc image as you can see in  the previous comments. No progress AFAIK in uDocker in this respect.

**Comment by OndraZizka** [comment]
Hi, seeking a little help here. Is your observation that solutions like `alpine-pkg-glibc` won't fix this kind of issues?
> Error relocating ...: __snprintf_chk: symbol not found

**Comment by gmolto** [comment]
Looks like version 1.1.3 of udocker provides some experimental code for Alpine according to: https://github.com/indigo-dc/udocker/issues/76

**Comment by rethab** [comment]
How can I try scar with version 1.1.3? I tried to modify the Dockerfile (and monkey-patched some files in the repo), but I can't get past this error:
```
Unable to find "/scar/src/providers/aws/cloud/lambda/udocker/udocker-1.1.3.tar.gz" when adding binary and data files.

The command '/bin/sh -c pyinstaller --onefile   --add-data="src/providers/aws/cloud/lambda/scarsupervisor.py:src/providers/aws/cloud/lambda"   --add-data="src/providers/aws/cloud/lambda/clients/apigateway.py:src/providers/aws/cloud/lambda/clients"   --add-data="src/providers/aws/cloud/lambda/clients/batch.py:src/providers/aws/cloud/lambda/clients"   --add-data="src/providers/aws/cloud/lambda/clients/lambdafunction.py:src/providers/aws/cloud/lambda/clients"   --add-data="src/providers/aws/cloud/lambda/clients/s3.py:src/providers/aws/cloud/lambda/clients"   --add-data="src/providers/aws/cloud/lambda/clients/udocker.py:src/providers/aws/cloud/lambda/clients"   --add-data="src/exceptions.py:src"   --add-data="src/utils.py:src"   --add-data="src/providers/aws/cloud/lambda/udocker/udocker-1.1.3.tar.gz:src/providers/aws/cloud/lambda/udocker"   --add-binary="udockerb:src/providers/aws/cloud/lambda/udocker"   --add-binary="/usr/bin/zip:src/bin"   --hidden-import=urllib3   --hidden-import=configparser   scar.py' returned a non-zero code: 1
```

**Comment by alpegon** [comment]
I created a branch called 'udocker-update' with the latest version of udocker.
This version solves the issue with alpine containers (at least the containers that I tested).
Please check the if branch works correctly with your containers.

**Comment by alpegon** [comment]
Fixed in PR #286.
Thanks to the new version of udocker the alpine containers can be used in AWS Lambda.
